ntbs-service/Pages/Notifications/OverviewPartials/_TreatmentEventsOverviewPartial.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@using ntbs_service.Helpers
+@using ntbs_service.Models.Enums
+@model OverviewModel
+
+@{
+    var sectionSubPath = NotificationSubPaths.EditTreatmentEvents;
+}
+
+<div class="notification-overview-type-and-edit-container" id="@OverviewSubPathToAnchorMap.GetOverviewAnchorId(sectionSubPath)">
+    <h3 class="notification-details-type"> Episodes </h3>
+    @if (Model.PermissionLevel == PermissionLevel.Edit)
+    {
+        <a href="@RouteHelper.GetNotificationPath(Model.NotificationId, sectionSubPath)"
+           class="notification-edit-link govuk-link--no-visited-state">
+            Edit
+        </a>
+    }
+</div>
+<div class="notification-overview-details-container">
+    <div>
+        @if (@Model.OutcomeAt12Month != null)
+        {
+            <div>
+                <strong> Outcome at 12 months </strong> @Model.OutcomeAt12Month.TreatmentOutcomeType.GetDisplayName() 
+            </div>
+        }
+        
+        @if (@Model.OutcomeAt24Month != null)
+        {
+            <div>
+                <strong> Outcome at 24 months </strong> @Model.OutcomeAt24Month.TreatmentOutcomeType.GetDisplayName()
+            </div>
+        }
+        
+        @if (@Model.OutcomeAt36Month != null)
+        {
+            <div>
+                <strong> Outcome at 36 months </strong> @Model.OutcomeAt36Month.TreatmentOutcomeType.GetDisplayName() 
+            </div>
+        }
+    </div>
+
+    @for (var episodeNumber = 1; episodeNumber <= Model.GroupedTreatmentEvents.Count; episodeNumber++)
+    {
+        <nhs-table is-with-panel="true" title-text="Episode @episodeNumber" caption="Other possible causes of your symptoms">
```

Comment:
Caption seems a bit suspect here, assuming a copy and paste from somewhere else?